Lib/subprocess.py

```diff
@@ -1490,8 +1490,9 @@ def _execute_child(self, args, executable, preexec_fn, close_fds,
                     if not comspec:
                         system_root = os.environ.get('SystemRoot', '')
                         comspec = os.path.join(system_root, 'System32', 'cmd.exe')
-                        if not os.path.isabs(comspec) or not os.path.isfile(comspec):
-                            raise FileNotFoundError('cmd.exe not found; neither %ComSpec% nor %SystemRoot% is set')
+                        if not os.path.isabs(comspec):
+                            raise FileNotFoundError(
+                'shell not found: neither %ComSpec% nor %SystemRoot% is set')
```

Comment:
```suggestion
                            raise FileNotFoundError('shell not found: neither %ComSpec% nor %SystemRoot% is set')
```

This is better all on one line. (I know PEP 8 says to split it, but it's okay to ignore it this time - there's no important code in or after the message, and it's more readable this way)

Reply:
@eryksun

Reply:
That's fine by me. I was trying to comply with the 79 character limit in PEP 8 while at least keeping the message itself all on one line.